doonut-core/src/main/resources/db/migration/V20240501234456__calender_ddl.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE IF NOT EXISTS calender
+(
+    id                BIGINT AUTO_INCREMENT               NOT NULL COMMENT 'ID',
+    member_id         BIGINT                              NOT NULL COMMENT '멤버 ID',
+    calender_name     VARCHAR(100)                        NOT NULL COMMENT '캘린더 명',
+    total_count       INT                                 NOT NULL COMMENT '총 기록 횟수',
```

Comment:
여기도 DEFAULT 값 추가해야할 것 같아요.

```suggestion
    total_count       INT       DEFAULT 0                 NOT NULL COMMENT '총 기록 횟수',
```